tools/scripts/upload_alerts_to_aws.py

```diff
@@ -54,16 +54,18 @@ def get_recent_alerts(orgname, reponame):
                                                         parameters=query_parameters)
     return api_response["results"]
 def merge_alerts(current_alerts, new_alerts):
+    merged_alerts = []
     current_alert_keys = set()
```

Comment:
Nit: May be rename `current_alert_keys` to `alert_keys` or `merged_alert_keys` as this set now keeps track of the list of merged alerts and not `current_alerts`